src/cascadia/TerminalApp/TerminalTab.cpp

```diff
@@ -788,7 +788,7 @@ namespace winrt::TerminalApp::implementation
         closeTabMenuItem.Click([weakThis](auto&&, auto&&) {

```

Comment:
do we need to do the same for Settings

Reply:
Settings inherit it from TabBase that already uses the correct event :blush: